CHANGELOG.md

```diff
@@ -1,11 +1,19 @@
-# Version 2.3.0
+# Version 3.0.0
 
-## [2.3.0]
+## [3.0.0]
 #### Added
--
+- ForgeRock iOS SDK now supports and available through Swift Package Manager. [SDKS-912]
+- ForgeRock iOS SDK now supports Social Login for `Sign in with Apple`, `Google Sign-In`, and `Facebook Login`.  [SDKS-879]
+- New SDK modules, `FRGoogleSignIn` and `FRFacebookSignIn`, are now available to enable Social Login with AM using providers' native SDKs. [SDKS-879]
+- `WebAuthnRegistrationCallback`, and `WebAuthnAuthenticationCallback` are introduced to support AM's `WebAuthn Registration Node` and `WebAuthn Authentication Node`. [SDKS-782]
+
```

Comment:
Apologies I should have added that in my branch, can we add the 
'Exposing Revoke Access Token method to FRUser' change